librato-iOS/Categories/NSString+AYLSanitizedForMetric.m

```diff
@@ -0,0 +1,21 @@
+//
+//  NSString+AYLSanitizedForMetric.m
+//  Librato-iOS
+//
+//  Created by Adam Yanalunas on 10/3/13.
+//  Copyright (c) 2013 Amco International Education Services, LLC. All rights reserved.
+//
+
+#import "NSString+AYLSanitizedForMetric.h"
+
+@implementation NSString (AYLSanitizedForMetric)
+
+- (NSString *)ayl_sanitizedForMetric
```

Comment:
Why not something like `lib_santizedForMetric` (Librato) or `lm_sanitizedForMetric` (Librato Metric) or `alm_santizedForMetric` (Amco Librato Metric)?